script/translations/migrate.py

```diff
@@ -327,12 +327,25 @@ def find_frontend_states():
 def run():
     """Migrate translations."""
     # Import new common keys
-    migrate_project_keys_translations(
+    rename_keys(
```

Comment:
What's happening here is that we one-time rename these keys to the common keys. Then when strings are uploaded after merge, the old names are created again, but their new value is set up as a reference to the new name.

Reply:
So this script needs to run before merge.

Reply:
Merge of this PR?

Reply:
Yes, I will do that once CI passses.